packages/medusa/src/api/routes/admin/collections/index.ts

```diff
@@ -69,12 +69,77 @@ export const defaultAdminCollectionsFields = [
 ]
 export const defaultAdminCollectionsRelations = ["products"]
 
+/**
+ * @schema AdminCollectionsListRes
+ * type: object
+ * properties:
+ *   collections:
+ *      type: array
+ *      items:
+ *        $ref: "#/components/schemas/ProductCollection"
+ *   count:
+ *      type: integer
+ *      description: The total number of items available
+ *   offset:
+ *      type: integer
+ *      description: The number of items skipped before these items
+ *   limit:
+ *      type: integer
+ *      description: The number of items per page
+ */
 export type AdminCollectionsListRes = PaginatedResponse & {
   collections: ProductCollection[]
 }
 
+/**
+ * @schema AdminCollectionsDeleteRes
+ * type: object
+ * properties:
+ *   id:
+ *     type: string
+ *     description: The ID of the deleted Collection
+ *   object:
+ *     type: string
+ *     description: The type of the object that was deleted.
+ *     default: product-collection
+ *   deleted:
+ *     type: boolean
+ *     description: Whether the collection was deleted successfully or not.
+ *     default: true
+ */
 export type AdminCollectionsDeleteRes = DeleteResponse
 
+/**
+ * @schema AdminDeleteProductsFromCollectionRes
+ * type: object
+ * properties:
+ *   id:
+ *     type: string
+ *     description: "The ID of the collection"
+ *   object:
+ *     type: string
+ *     description: "The type of object the removal was executed on"
+ *     default: product-collection
+ *   removed_products:
+ *     description: "The IDs of the products removed from the collection"
+ *     type: array
+ *     items:
+ *       description: "The ID of a Product to add to the Product Collection."
+ *       type: string
+ */
+export type AdminDeleteProductsFromCollectionRes = {
+  id: string
+  object: string
+  removed_products: string[]
+}
```

Comment:
`admin/collections/remove-products.ts` response shape had not equivalent class declared. 

Our JS client is wrongfully typed with `AdminCollectionsDeleteRes`. See [file](https://github.com/medusajs/medusa/blob/3ca6e7f8c494187f1c610cd076ba0b01efb56c82/packages/medusa-js/src/resources/admin/collections.ts#L115-L123).